.rubocop.yml

```diff
@@ -2,3 +2,7 @@ inherit_gem:
   rubocop-github:
     - config/default.yml
     - config/rails.yml
+
+AllCops:
+  Exclude:
+    - "test/tmp/**/*"
```

Comment:
should we .gitignore this too?

Reply:
Good point! After the test the directory gets cleaned up, but doesn't hurt to add it to .gitignore too 🙂